docs/tutorials/_shared/setup-vs2022/01-create-project.md

```diff
@@ -12,6 +12,11 @@ Name the project any way you want:
 
 Choose the _.NET 6.0 (Long-term support)_ framework, and disable HTTPS as we don't need it for now:
 
+:::caution
+Disable the use of top-level statements if it appears on the `Additional Information` window.
+:::
```

Comment:
The wording makes it sound like we're asking the reader to disable top-level statements which is exactly the opposite of what we want to do :) I would suggest something like `We are using top-level statements in our examples, so if you see a checkbox saying "Don't use top-level statements", make sure it is unchecked.`.

Reply:
@Arrotech have you had a chance to follow up on this?